app/build.gradle.kts

```diff
@@ -66,4 +73,94 @@ dependencies {
     androidTestImplementation(libs.androidx.ui.test.junit4)
     debugImplementation(libs.androidx.ui.tooling)
     debugImplementation(libs.androidx.ui.test.manifest)
-}
\ No newline at end of file
+
+    // Hilt - dependency injection
+    implementation(libs.hilt.android)
+    // hilt kapt - annotation processor — hilt ksp is in alpha
+    kapt(libs.hilt.compiler)
+
+    // Retrofit - make type safe requests
+    implementation(libs.retrofit)
+    // okhttp - fire http calls
+    implementation(libs.okhttp)
+    // logging interceptor - log calls for debugging
+    implementation(libs.logging.interceptor)
+    // converter - make retrofit use serialization json
+    implementation(libs.converter.kotlinx.serialization)
+    // JSON - serialize Kotlin objects <-> JSON
+    implementation(libs.kotlinx.serialization.json)
+
+    // Coil - asynchronous image loading
+    implementation(libs.coil.compose)
+
+    // datastore - make type safe local storage
+    implementation(libs.androidx.datastore)
+    // protobuf - serialize Kotlin objects <-> protobuf schema
+    implementation(libs.protobuf.kotlin.lite)
+
```

Comment:
Can you explain what is this for?

Reply:
Protobuf is a language-neutral mechanism for serializing structured data. It supports many languages.
Here we choose the required language with
```
register("java") { option("lite") }
register("kotlin") { option("lite") }
```
and add the needed dependency for it with 

```
implementation(libs.protobuf.kotlin.lite)
```
Please, let me know if you'd like me to elaborate further.

Reply:
I think a better comment for it could be
```
// protobuf - make Kotlin objects based on protobuf schema
```

Reply:
where do you think we will need it?

Reply:
The proto datastore needs a `Serializer<T>` and in our case the type `T` will be the generated Java class.
The `Serializer` determines the on-disk format and API for accessing it.
The type `T` handles encoding and decoding the data.

If it is too much for our use cases we can use kotlinx serialization json 
or even drop it all together and use preferences datastore